kotlinx-coroutines-core/build.gradle

```diff
@@ -183,6 +190,14 @@ jvmTest {
         exclude '**/*StressTest.*'
     }
     systemProperty 'kotlinx.coroutines.scheduler.keep.alive.sec', '100000' // any unpark problem hangs test
+
+    // TODO: JVM IR generates different stacktrace so temporary disable stacktrace tests
```

Comment:
Do I have to set up a reminder for that? When is it safe to remove this filter?

Reply:
@udalov

Reply:
I suppose when we switch kotlinx-coroutines to JVM IR, we'll just update these stacktraces with the new expectations and unmute the tests. Stacktraces differ slightly in JVM IR because of slightly different logic in the tail call optimization. Here's the commit that makes the tests pass for me on JVM IR, @qwwdfsad please verify that it's OK (to be applied later when we enable JVM IR) or if there's something that needs to be fixed in the backend:
https://github.com/udalov/kotlinx.coroutines/commit/352a173dd160de3ceaa1266c4eeb45cb7d94fbb6

BTW @ALikhachev `CoroutinesDumpTest` passes for me with JVM_IR, so probably it can be unmuted (both here and in -debug).

Reply:
Verified 👍 

@ALikhachev please unmute `CoroutinesDumpTest` then